x-pack/docs/en/security/auditing/event-types.asciidoc

```diff
@@ -6,132 +6,390 @@ When you are <<enable-audit-logging,auditing security events>>, a single client
 might generate multiple audit events, across multiple cluster nodes. The common
 `request.id` attribute can be used to correlate the associated events.
 
-|======
-| Event action | Description | Example
-
-| `access_denied`
-| Logged when an authenticated user attempts to execute an action they do not
+[[event-access-denied]]
+`access_denied`::
+Logged when an authenticated user attempts to execute an action they do not
 have the necessary <<security-privileges,privilege>> to perform.
-| `{"type":"audit", "timestamp":"2020-12-30T22:30:06,949+0200", "node.id":"0RMNyghkQYCc_gVd1G6tZQ", "event.type":"transport", "event.action":"access_denied", "authentication.type":"REALM", "user.name":"user1", "user.realm":"default_native", "user.roles":["test_role"], "origin.type":"rest", "origin.address":"[::1]:52434", "request.id":"yKOgWn2CRQCKYgZRz3phJw", "action":"indices:admin/auto_create", "request.name":"CreateIndexRequest", "indices":["<index-{now/d+1d}>"]}`
-
-| `access_granted`
-| Logged when an authenticated user attempts to execute an action they have the
++
+.Example
+[%collapsible%open]
+====
+[source,js]
+{"type":"audit", "timestamp":"2020-12-30T22:30:06,949+0200", "node.id":
+"0RMNyghkQYCc_gVd1G6tZQ", "event.type":"transport", "event.action":
+"access_denied", "authentication.type":"REALM", "user.name":"user1", 
+"user.realm":"default_native", "user.roles":["test_role"], "origin.type":
+"rest", "origin.address":"[::1]:52434", "request.id":"yKOgWn2CRQCKYgZRz3phJw",
+"action":"indices:admin/auto_create", "request.name":"CreateIndexRequest", 
+"indices":["<index-{now/d+1d}>"]}
+====
+
+[[event-access-granted]]
+`access_granted`::
+Logged when an authenticated user attempts to execute an action they have the
 necessary privilege to perform. When the `system_access_granted` event is
```

Comment:
Should we list `system_access_granted` as any other event type in this list? We can still cross-reference it in this description, but I think it's clearer if we provide the full list as expected. I personally got confused by the sentence in the first place.

Reply:
Thanks @bytebilly. I added `system_access_granted` as an event type in a54d6bfcaec9f8ffa14ce907355a2fccf5e95b00 based on suggestions from @albertzaharovits.